src/api-service/__app__/agent_events/__init__.py

```diff
@@ -173,8 +179,21 @@ def on_worker_event(machine_id: UUID, event: WorkerEvent) -> None:
                     ],
                 )
             )
+            if task.config.debug and (
+                TaskDebugFlag.keep_node_on_failure in task.config.debug
+                or TaskDebugFlag.keep_node_on_completion in task.config.debug
+            ):
+                node.manual_reset_override = True
```

Comment:
Can you revisit the name of this property? I think `manual` and `override` throw me off. How about something in the ballpark of `debug_keep_node`, to tie it to the other new property names.